### PR TITLE
Turkish language support

### PR DESCRIPTION
**Pull request by YusufOzmen01**

### CODE REVIEW

**Comment by xou816** [comment]
Hey there, thank you very much for the contribution! :) I'll have a look a bit later!

File: po/tr.po [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-06-14 00:00+0200\n"
+"PO-Revision-Date: 2021-06-14 00:00+0200\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: Yusuf Çınar Özmen <marulumbenaq@gmail.com>\n"
+"Language: Turkish\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
```
Comment (ondras12345) [inline po/tr.po:15]: You need to fill in an expression here. `"Plural-Forms: nplurals=2; plural=(n != 1);\n"` should be correct for Turkish.
See https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html
Tools like [Poedit](https://poedit.net/) should do this for you automatically.

File: po/tr.po [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+msgstr ""
+"Project-Id-Version: spot\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-06-14 00:00+0200\n"
+"PO-Revision-Date: 2021-06-14 00:00+0200\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: Yusuf Çınar Özmen <marulumbenaq@gmail.com>\n"
+"Language: Turkish\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
```
Comment (ondras12345) [inline po/tr.po:13]: Replace `CHARSET` with `UTF-8`.

File: po/tr.po [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+msgid ""
+msgstr ""
+"Project-Id-Version: spot\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-06-14 00:00+0200\n"
+"PO-Revision-Date: 2021-06-14 00:00+0200\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: Yusuf Çınar Özmen <marulumbenaq@gmail.com>\n"
+"Language: Turkish\n"
```
Comment (ondras12345) [inline po/tr.po:11]: This needs to be `"Language: tr\n"`

File: po/tr.po [hunk 4]
```diff
@@ -0,0 +1,159 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
```
Comment (ondras12345) [inline po/tr.po:2]: Take a look at the [English translation](https://github.com/xou816/spot/blob/master/po/en.po) to see how to fill this in.
https://github.com/xou816/spot/blob/552ad4503180b5dcff5de8c5bb877c4e4afe5b68/po/en.po#L1-L4

**Review by YusufOzmen01 — COMMENTED** [review]
Title changed

**Comment by ondras12345** [comment]
Great! Looks good now.
I haven't tested it though, it seems like we need @xou816 to approve running the `spot-snapshots` workflow to get a build artifact. I don't feel like building it locally from source right now.

**Comment by xou816** [comment]
Thanks a lot Ondras, well both of you :) workflow should run!

**Comment by ondras12345** [comment]
Screenshot from 2021-06-14 14-45-19
Great! Seems to work just fine.

**Comment by YusufOzmen01** [comment]
> Screenshot from 2021-06-14 14-45-19
> Great! Seems to work just fine.

happy to see that :D

**Comment by xou816** [comment]
Looks good to me, merging! :)

**Pull request merged**